API/Services/Interfaces/ITournamentsService.cs

```diff
@@ -75,4 +75,32 @@ bool preApprove
     /// </summary>
     /// <param name="id">Tournament id</param>
     Task DeleteAsync(int id);
+
+    /// <summary>
+    /// Adds a collection of osu! beatmap ids to the tournament's PooledBeatmaps collection
+    /// </summary>
+    /// <param name="id">Tournament id</param>
+    /// <param name="osuBeatmapIds">A collection of osu! beatmap ids to add</param>
+    /// <returns>The tournament's pooled beatmaps</returns>
+    Task<ICollection<BeatmapDTO>> AddPooledBeatmapsAsync(int id, ICollection<long> osuBeatmapIds);
+
+    /// <summary>
+    /// Gets the <see cref="Tournament"/>'s <see cref="Tournament.PooledBeatmaps"/> collection
+    /// </summary>
+    /// <param name="id">Tournament id</param>
+    /// <returns>The <see cref="Tournament"/>'s <see cref="Tournament.PooledBeatmaps"/> collection</returns>
+    Task<ICollection<BeatmapDTO>> GetPooledBeatmapsAsync(int id);
```

Comment:
Should document that it will return an empty collection if the tournament does not exist